torch/_library/utils.py

```diff
@@ -103,6 +104,15 @@ def is_functional(schema):
     return is_functional(schema)
 
 
+def is_tensorlist_like_type(typ: torch.Type):
```

Comment:
I guess we have this in torchgen already? Do we expect deduplication to be able to happen between the two or they can't depend on each other?

Reply:
Ho? Which ones don't?

Reply:
resolved offline: long-term state is that we want deduplication but some internal builds make it difficult to deduplicate.